WISE-Community/WISE
# Added default projectLibraryGroups to db init script

This commit adds a default empty array string ("[]") for the portal.projectLibraryGroups field when the user creates the initial database and imports initial values.

How to test
1. Log into mysql and create a dummy database for testing, e.g. "wise_dummy"
```
mysql> create database wise_dummy;
```
2. From WISE_DIR directory, import the init database sql (src/main/resources/wise_db_init.sql) into the new wise_dummy database.
```
WISE_DIR $ mysql wise_dummy -u USERNAME -p < src/main/resources/wise_db_init.sql
```
3. Log into mysql and verify that portal.projectLibraryGroups field for ID=1 is populated with [].
```
mysql> select * from wise_dummy.portal where id=1;
```

Closes #1513
